spec/presenters/coronavirus_landing_page_presenter_spec.rb

```diff
@@ -192,13 +192,13 @@
               "heading" => "18 September",
               "national_applicability" => %w[england northern_ireland scotland wales],
               "paragraph" => "If you live, work or travel in the North East, you need to [follow different covid rules](/guidance/north-east-of-england-local-restrictions)\n",
-              "tags" => "<strong class='govuk-tag govuk-tag--blue covid-timeline__tag'>UK Wide</strong>",
+              "tags" => "<span class='govuk-visually-hidden'>, this guidance applies </span><strong class='govuk-tag govuk-tag--blue covid-timeline__tag'>UK Wide</strong>",
```

Comment:
Is there any scope to alter these to sentence case as well?  _[Capital letters are reputed to be 13 to 18% harder for users to read. So we try to avoid them.](https://www.gov.uk/guidance/content-design/writing-for-gov-uk)_

![image](https://user-images.githubusercontent.com/71266765/141776825-e5f6ac58-f324-4d2a-9ecc-715efad1e6ee.png)

Reply:
Do you mean changing `UK Wide` to `UK wide`?

Reply:
@chris-gds updated.

Reply:
> Do you mean changing UK Wide to UK wide? 

Ah either / or  (not entirely sure what the ideal Content rule is in this context)